### PR TITLE
Fix job character names

### PR DESCRIPTION
**Pull request by MrStonedOne**
I dont know what this pr is. I typed into a browser and this came out.

### CODE REVIEW

**Comment by Nabski89** [comment]
Hi and welcome to our little pet project. I'm absolutely in awe that you're here and wanting to contribute. We tried to use the person with the most recorded hours as provided in a simple tool from bobbahbrown to decide who was the skill guide for each role, then match their picture to the actual static. In this case Reed had 970 hours in botany over the next highest at 660. He also had 1042 hours as a chemist compared to the next highest as 405.

Basically this is a toss up between where we wanted to put him. 

If you could tell me who Daisy is and why they should be the replacement (and maybe point me to where I need to go to find a picture of their static) that would really help.

Depending on your answer we may also need to change engineering.

**Comment by Nabski89** [comment]
Completed in My 1.1 patch part 2 pull.
